sections/semantics-document-metadata.include

```diff
@@ -449,7 +449,7 @@
   The <dfn element-attr for="link"><code>media</code></dfn> attribute says which media the resource
   applies to. The value must be a <a>valid media query list</a>.
 
-  <!-- def <{link/integrity}> -->
+  The <dfn element-attr for="link"><code>integrity</code></dfn> attribute represents the <a>integrity metadata</a> for requests which this element is responsible for. The value is text. The attribute must not be specified on <{link}> elements that do not have a <{link/rel}> attribute that contains the <{link/stylesheet}> keyword. [[!SRI]]
```

Comment:
please split this line for easier reading in tools that don't auto-wrap. E.g. < 100 characters / line ...